### ISSUE TITLE
Question: multiple same-level aggregations?

### ISSUE THREAD

**Issue by zhangoose** [elastic/elasticsearch-dsl-py#1052]
Hi there! ✨ 

I'm trying to figure out how to write my query's aggregation block in `elasticsearch-dsl`. I've been looking around the issues for someone who's also trying to do multiple same-level aggregations with no luck, perhaps I've missed something?

### Query

```javascript
{
	"query": { .... }, // let's assume I already have this 
	"aggregations": {
		"all_aggregations": {
			"global": {},
			"aggregations": {
				"foos": {
					"filter": {
						"term": {
							"bar": "abc"
						}
					},
					"aggregations": {
						"filtered_foos": {
							"terms": {
								"field": "foo"
							}
						}
					}
				},
				"bars": {
					"filter": {
						"term": {
							"foo": "xyz"
						}
					},
					"aggregations": {
						"filtered_bars": {
							"terms": {
								"field": "bar"
							}
						}
					}
				}
			}
		}
	}
}
```

### Use case

([#1014 Faceted search improvements](https://github.com/elastic/elasticsearch-dsl-py/issues/1014) - I think I may be doing something similar to this)

Implementing faceted search on an ecommerce site. I'm trying to build the side-bar navigation using Elasticsearch aggregations (attached a gif for better visualizations):

[![img](https://duaw26jehqd4r.cloudfront.net/items/3s3w2V3K22251h2n2X3I/Screen%20Recording%202018-11-02%20at%2001.27%20PM.gif?v=977d16a5)](https://cl.ly/6a09185fb910)

Basically, each filter category should apply all filters except the ones related to that category. This concept is better described in this article: [Faceted search in a sidebar](https://madewithlove.be/faceted-search-using-elasticsearch/)

-------

Anyway, not sure if any of that context was helpful, but any assistance with this same-level aggregation translation to `elasticsearch-dsl` would be super appreciated! 🌱

**Comment by HonzaKral** [comment]
Hi, this is exactly what the `FacetedSearch` class is doing so I would refer you to inspect the code (0), tests (1), and documentation (2). Thank you!

0 - https://github.com/elastic/elasticsearch-dsl-py/blob/master/elasticsearch_dsl/faceted_search.py
1 -  https://github.com/elastic/elasticsearch-dsl-py/blob/master/test_elasticsearch_dsl/test_faceted_search.py
2 - https://elasticsearch-dsl.readthedocs.io/en/latest/faceted_search.html#example
